component/vdr/indy/creator.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright Scoir Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package indy
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/btcsuite/btcutil/base58"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+	vdrapi "github.com/hyperledger/aries-framework-go/pkg/framework/aries/api/vdr"
+)
+
+// Build builds a new DID Doc.
+func (r *VDR) Build(pubKey *vdrapi.PubKey, opts ...vdrapi.DocOpts) (*did.Doc, error) {
```

Comment:
How does this DID get written to the Indy network?

Reply:
The assumption is that applications will write DIDs to the ledger outside of the VDR using the wrapper for lib-indyvdr directly.